### DISCUSSION TITLE
Can you detect if a tour is already playing?

### DISCUSSION THREAD

**Discussion by MincDev** [hakimio/ngx-ui-tour#68] [General]
Hi, is there a way to determine whether a tour is already playing? Say for example I want to start another tour, but I don't want to do it while a current tour is playing as they overlap. How can I detect that a tour is already underway?

**Comment by hakimio** [comment]
```ts
this.tourService.getStatus() === TourState.ON
```

**Reply by MincDev** [reply]
Perfect! Thanks for the prompt response!
